napari/layers/transforms.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import annotations  # noqa: F407
```

Comment:
Note: this is valid in Python 3.7 but PyFlakes doesn't seem to think so... I don't really understand why:

https://github.com/PyCQA/pyflakes/pull/374